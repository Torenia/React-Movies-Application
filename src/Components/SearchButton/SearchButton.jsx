import styled from 'styled-components';

export const SearchButton = styled.div`
  position: absolute;
  top: 25px;
  right: 165px;
  
  i {
    color: red;
    font-size:24px
  }
`;
