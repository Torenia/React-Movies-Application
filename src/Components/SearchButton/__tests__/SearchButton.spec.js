import React from 'react';

import { SearchButton } from '../SearchButton';

describe('<SearchButton />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SearchButton />);
  });

  it('should render a search button', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
