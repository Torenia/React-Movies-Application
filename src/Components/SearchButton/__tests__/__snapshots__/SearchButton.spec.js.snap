// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchButton /> should render a search button 1`] = `
.c0 {
  position: absolute;
  top: 25px;
  right: 165px;
}

.c0 i {
  color: red;
  font-size: 24px;
}

<div
  className="c0"
/>
`;
